client/app/scripts/superdesk-highlights/highlights.js

```diff
@@ -206,9 +207,21 @@
                 }
                 highlightsService.clearCache();
                 $scope.modalActive = false;
-            }, function(result) {
-                $scope.message = gettext('There was a problem while saving configuration');
+            }, function(response) {
+                errorMessage(response);
             });
+            
+            function errorMessage(response) {
+                if (response.data && response.data._issues && response.data._issues.name && response.data._issues.name.unique) {
+                    $scope.message = gettext(
```

Comment:
this can't be translated, it's creating different string ids for each config name